### ISSUE TITLE
make auto-correct work with trailing comments on a method

### ISSUE THREAD

**Issue by jonknapp** [CoffeeAndCode/rubocop_method_order#1]
I'm not sure we handle comments that immediately follow a method, so the following example may fail to be auto-corrected as expected:

```ruby
# rubocop:disable SomeRule
def bear
end
# rubocop:enable SomeRule

def apple
end
```

The expected output would be:

```ruby
def apple
end

# rubocop:disable SomeRule
def bear
end
# rubocop:enable SomeRule
```

**Comment by jonknapp** [comment]
Fixed in https://github.com/CoffeeAndCode/rubocop_method_order/commit/028aa7c69ed461d9badef45f6df602c6020bb889, currently unreleased.

**Comment by jonknapp** [comment]
Released in v0.3.0
